testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/interceptor/serverside/ServerInterceptor.java

```diff
@@ -0,0 +1,22 @@
+package org.jboss.as.test.integration.ejb.interceptor.serverside;
+
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.InvocationContext;
+import org.jboss.logging.Logger;
+
+/**
+ * Server side sample interceptor
+ */
+public class ServerInterceptor {
+
+    public ServerInterceptor() {
+    }
+
+    private Logger log = Logger.getLogger(ServerInterceptor.class.getName());
+
+    @AroundInvoke
+    public Object aroundInvoke(final InvocationContext invocationContext) throws Exception {
+        log.debug("serverInterceptor aroundInvoke ");
```

Comment:
We need to check that the interceptor was executed.

Reply:
Added checking with CountDownLatch